doc/cascadia/JSONFragments.md

```diff
@@ -0,0 +1,123 @@
+# JSON fragment extensions in Windows Terminal
+
+JSON fragment extensions are snippets of JSON that application developers can write to add new profiles to users' settings, or even modify certain existing profiles. They can also be used to add new color schemes to users' settings.
+
+## Structure of the JSON files 
+
+The JSON file should be split up into 2 lists, one for profiles and one for schemes. Here is an example of a json file that adds a new profile, modifies an existing profile, and creates a new color scheme: 
+
+```JSON 
+{
+    "profiles": [
+        {
+            "updates": "{2c4de342-38b7-51cf-b940-23e9ae97f518}",
+            "fontSize": 16,
+            "fontWeight": "thin"
+        },
+        {
+            "name": "Cool Profile",
+            "commandline": "powershell.exe",
+            "antialiasingMode": "aliased",
+            "fontWeight": "bold",
+            "colorScheme": "Postmodern Tango Light"
+        }
+    ],
+    "schemes": [
+        {
+            "name": "Postmodern Tango Light",
+
+            "black": "#0C0C0C",
```

Comment:
This feels like a nit, but should the colors be in RGB order, rather than alphabetical? I'd think that most people editing a terminal color scheme think of the palette in the terminal ordering, and this is code that's going to be copy-pasted 1000 times in the future